README.md

```diff
@@ -105,6 +106,20 @@ aggregate?: (expression: Alg.AggregateExpression) => Promise<RDF.Term>;
 
 You can probably ignore this.
 
+### Extension functions
+
+Extension functions can be added by providing the `extensionFunctionCreator` in the config.
+Example
+```ts
+config.extensionFunctionCreator = (functionName: RDF.NamedNode) => {
+   if (functionNamedNode.value === 'https://example.org/functions#equal') {
```

Comment:
This may not be the most interesting example.
Could you add one for uppercasing a string instead?
(would then also require checking if the term is a literal)

Reply:
Since this would be hard to test as of right now maybe we should leave this example as is? We can update it when we can also test this example?

Reply:
Sounds good!